portal/templates/redesign/teach_new/materials_new.html

```diff
@@ -81,15 +83,15 @@ <h1 class="banner__text--primary">Teaching Resources</h1>
             <div class="row">
                 <div class="col-sm-6">
                     <h4>Key Stage 1, and Lower Key Stage 2</h4>
-                    <p class="body-text"><a>Introduction to Coding Guide</a></p>
-                    <p class="body-text"><a>Levels Guide (1-50)</a></p>
-                    <p class="body-text"><a>Glossary</a></p>
+                    <p class="body-text"><a id="pdf_link" href= "{% url 'materials_viewer_new' pdf_name='introduction_to_coding' %}">Introduction to Coding Guide</a></p>
```

Comment:
PDF etiquette: It's a good idea to warn people that the link goes to something that's not a web page.  Back in the days of dialup, we used to add the filesize as well, but that's less important now.  Maybe talk to James about adding an icon or a note, or something?